go.mod

```diff
@@ -29,12 +29,12 @@ require (
 	github.com/strangelove-ventures/packet-forward-middleware/v5 v5.1.2
 	github.com/stretchr/testify v1.8.4
 	github.com/syndtr/goleveldb v1.0.1-0.20220721030215-126854af5e6d
-	github.com/tendermint/tendermint v0.34.28
+	github.com/tendermint/tendermint v0.34.29
```

Comment:
I noticed the other day we are on .29 anyway by virtue of the replace; but nice to have them match.